### ISSUE TITLE
onSlideNextStart events getting override when i click on the bootstrap tab which was inactive then coming back to previous one

### ISSUE THREAD

**Issue by Feroz-Siddiqui** [nolimits4web/Swiper#1760]
Initially swiper in bootstrap were not working. then i call update /resize function on tab click. everything was working fine. after putting alert in onSlideNextStart function i get to know that after calling update/resize function mine swiper1 onSlideNextStart function is not getting called when i switch from home tab to profile tab and then came back to profile tab to home tab.

<!DOCTYPE html>

Example of Bootstrap 3 Tabs Events
href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css">
src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js">
src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js">

body { background: #eee; font-family: Helvetica Neue, Helvetica, Arial, sans-serif; font-size: 14px; color: #000; margin: 0; padding: 0; overflow-y: hidden; overflow-x: hidden; } .swiper-container { width: 500px; height: 300px; margin: 20px auto; } .swiper-slide { text-align: center; font-size: 18px; background: #fff; /* Center slide text vertically */ display: -webkit-box; display: -ms-flexbox; display: -webkit-flex; display: flex; -webkit-box-pack: center; -ms-flex-pack: center; -webkit-justify-content: center; justify-content: center; -webkit-box-align: center; -ms-flex-align: center; -webkit-align-items: center; align-items: center; } .bs-example { margin: 20px; }

Home

Profile
    </ul>
    <div class="tab-content" id="myTabContent">
        <div id="sectionA" class="tab-pane fade in active">
            <div class="swiper-container swiper1">
                <div class="swiper-wrapper">
                    <div class="swiper-slide">Slide 1</div>
                    <div class="swiper-slide">Slide 2</div>
                    <div class="swiper-slide">Slide 3</div>
                    <div class="swiper-slide">Slide 4</div>
                    <div class="swiper-slide">Slide 5</div>
                    <div class="swiper-slide">Slide 6</div>
                    <div class="swiper-slide">Slide 7</div>
                    <div class="swiper-slide">Slide 8</div>
                    <div class="swiper-slide">Slide 9</div>
                    <div class="swiper-slide">Slide 10</div>
                </div>
                
                <div class="swiper-pagination swiper-pagination1"></div>
                 
    <div class="swiper-button-next swiper-button-next1"></div>
    <div class="swiper-button-prev swiper-button-prev1"></div>
            </div>
        </div>
        <div id="profile" class="tab-pane fade">
            
            <div class="swiper-container swiper2">
                <div class="swiper-wrapper">
                    <div class="swiper-slide">Slide 1</div>
                    <div class="swiper-slide">Slide 2</div>
                    <div class="swiper-slide">Slide 3</div>
                    <div class="swiper-slide">Slide 4</div>
                    <div class="swiper-slide">Slide 5</div>
                    <div class="swiper-slide">Slide 6</div>
                    <div class="swiper-slide">Slide 7</div>
                    <div class="swiper-slide">Slide 8</div>
                    <div class="swiper-slide">Slide 9</div>
                    <div class="swiper-slide">Slide 10</div>
                </div>
                
                <div class="swiper-pagination swiper-pagination2"></div>
    
    <div class="swiper-button-next swiper-button-next2"></div>
    <div class="swiper-button-prev swiper-button-prev2" ></div>
            </div>

        </div>
        <hr>
        <p class="active-tab">
            <strong>Active Tab</strong>: <span></span>
        </p>
        <p class="previous-tab">
            <strong>Previous Tab</strong>: <span></span>
        </p>
    </div>
    <script src="css/swiper.min.js"></script>

    <script>
var swiper1 = new Swiper('.swiper1', {

pagination: '.swiper-pagination1',
paginationClickable: true,
spaceBetween: 30,
nextButton: '.swiper-button-next1',
prevButton: '.swiper-button-prev1',
/*continous swiping*/
loop:true,
onSlideNextStart : function(swiper) {
    alert('swiper1 next');
},onSlidePrevStart : function(swiper) {
    alert('swiper1 prev');  
}
});
var swiper2 = new Swiper('.swiper2', {
pagination: '.swiper-pagination2',
paginationClickable: true,
spaceBetween: 30,
nextButton: '.swiper-button-next2',
prevButton: '.swiper-button-prev2',
/continous swiping/
loop:true,
onSlideNextStart : function(swiper) {
alert('swiper2 next');
},onSlidePrevStart : function(swiper) {
alert('swiper2 prev');

}

});

$('a[data-toggle="tab"]').on('shown.bs.tab', function(e){
var activeTab = $(e.target).text(); // Get the name of active tab
var previousTab = $(e.relatedTarget).text(); // Get the name of previous tab
alert(activeTab);
swiper2.update();
});

/* jQuery('.swiper-container').each(function(){
mySwiper = jQuery(this).swiper({
onSlideNextStart : function(swiper) {
alert($(this).html());
}
});
}); / 
/ var swiper = new Swiper('.aaaa');
var swiper1= new Swiper('.bbbb');
*/

**Comment by nolimits4web** [comment]
If it still actual, create new issue and provide more details
